build/includes/release.mk

```diff
@@ -65,36 +65,14 @@ release-deploy-site:
 	echo "Deploying Site Version: $$version" && \
 	$(MAKE) ENV=HUGO_ENV=snapshot site-deploy SERVICE=$$version
 
-# Creates a release. Version defaults to the base_version
-# - Checks out a release branch
-# - Build binaries and images
-# - Creates sdk and binary archives, and moves the into the /release folder for upload
-# - Creates a zip of the install.yaml, LICENCE and README.md for installation
-# - Pushes the current chart version to the helm repository hosted on gcs.
-do-release: RELEASE_VERSION ?= $(base_version)
-do-release: $(ensure-build-image)
-	@echo "Starting release for version: $(RELEASE_VERSION)"
-
-	# switch to the right project
-	$(DOCKER_RUN) gcloud config configurations activate agones-images
-
-	git checkout -b release-$(RELEASE_VERSION)
-	-rm -rf $(agones_path)/release
-	mkdir $(agones_path)/release
-
-	$(MAKE) -j 4 build VERSION=$(RELEASE_VERSION) REGISTRY=$(release_registry) FULL_BUILD=1
-	cp $(agones_path)/cmd/sdk-server/bin/agonessdk-server-$(RELEASE_VERSION).zip $(agones_path)/release
-	cp $(agones_path)/sdks/cpp/.archives/agonessdk-$(RELEASE_VERSION)-linux-arch_64.tar.gz $(agones_path)/release
-	cd $(agones_path) &&  zip -r ./release/agones-install-$(RELEASE_VERSION).zip ./README.md ./install ./LICENSE
-
-	$(MAKE) gcloud-auth-docker
-	$(MAKE) -j 4 push REGISTRY=$(release_registry) VERSION=$(RELEASE_VERSION)
```

Comment:
Let's keep the target, but remove the building of the images and pushing them.

We still need to push the helm chart (which is being worked on), and also work out where to sort out the git branch work as well.

Reply:
Included the do-release target with push-chart and git branch. Will work on the branch part along with push-chart target in b/278275965